core/tests/protractor_utils/general.js

```diff
@@ -51,6 +51,12 @@ var CONSOLE_ERRORS_TO_IGNORE = [
   _.escapeRegExp(
     'http://localhost:9099/www.googleapis.com/identitytoolkit/v3/' +
     'relyingparty/verifyPassword?key=fake-api-key'),
+  // These "pencilcodeembed" erros occurs when the pencil code site tries to
```

Comment:
This error covers the case when the PencilCode site uses an invalid SSL certificate (which can happen when it expires). In such cases, we ignore the error since it is out of our control.

Reply:
Done